lib/models/component-property-controls.ts

```diff
@@ -217,6 +218,15 @@ export interface ComponentPropertyControlInteractive {
     viewLink: string;
 }
 
+export interface ComponentPropertyControlSlider {
```

Comment:
Some other controls seem to have documentation per property, might be nice for this one as well?

Reply:
I do not, it 's a generic property. And it has quite obvious param names

Reply:
> Code is ok I guess, not really sure if this contains all the required changes? I guess it should also be listed in a new components schema?

I am not sure I have to do it all together, the minimum needed is a new property type to be able to use in in DE

Maybe new components schema could be added in a separate PR?

Reply:
Discussed a bit this property type with Sander https://github.com/WoodWing/contentstation/pull/2169#discussion_r595915134
And it's not clear for now should be `defaultValue` mandatory or optional. And what behavior is expected if value is `undefined`.
Looks like this topic needs to be discussed with UX team